test/integration/document/Thumbnails.e2e.test.js

```diff
@@ -64,7 +64,7 @@ describe('Preview Document Thumbnails', () => {
      * @return {Assertion} Chai Assertion
      */
     const notBeVisible = ($thumbnailsSidebar) =>
-        expect($thumbnailsSidebar).to.have.css('transform', 'matrix(1, 0, 0, 1, -201, 0)'); // translateX(-201px)
+        expect($thumbnailsSidebar).to.have.css('transform', 'matrix(1, 0, 0, 1, -226, 0)'); // translateX(-226px)
```

Comment:
Should we just check for transform? Then we don't need to worry about a change in value.

Reply:
What I want to verify is that the transform value changed from 0 to 226 or vice versa, to reflect the state of the sidebar toggling